### DISCUSSION TITLE
Mobile version

### DISCUSSION THREAD

**Discussion by Chrisdeparis** [community/community#120129] [Mobile]
Why the pictures of my repo don't appear on the mobile version ? When i add images to a markdown file the images don't show.

**Comment by stevepopovich** [comment]
Hi @Chrisdeparis do you have an example screenshot of this?

**Reply by Chrisdeparis** [reply]
Yes this is what i see on mobile app ![Screenshot_2024-04-19-10-57-51-599_com.github.android.jpg](https://github.com/community/community/assets/9371437/f0ebbf3d-37cb-40f9-9317-1c9d396ade5d)

**Reply by stevepopovich** [reply]
Interesting, is this a private repo? We definitely expect this to work, thanks for the screenshot

**Reply by Chrisdeparis** [reply]
Yes it's a private repo.

**Reply by stevepopovich** [reply]
Ok. Thanks for writing in. I logged this with the team to take a look.

**Comment by Chrisdeparis** [comment]
Yes it's a private repo.

Le ven. 19 avr. 2024, 16:35, Steven Popovich ***@***.***> a
écrit :
